### ISSUE TITLE
missing documentation: configure healthcheck (timeouts) for components

### ISSUE THREAD

**Issue by nathansamson** [deis/workflow#725]
I noticed that my builder component is quite often failing its health check (which results in restarts as well)

~~~
Events:
  FirstSeen	LastSeen	Count	From			SubobjectPath			Type		Reason		Message
  ---------	--------	-----	----			-------------			--------	------		-------
  1d		1m		131	{kubelet 10.129.22.70}	spec.containers{deis-builder}	Warning		Unhealthy	Liveness probe failed: Get http://10.2.25.9:8092/healthz: net/http: request canceled (Client.Timeout exceeded while awaiting headers)
~~~

This might be related to deis/builder#180

To fix this I wanted to increase the timeout, but I don't see any specific documentation on how to do so
(I would expect it here: https://deis.com/docs/workflow/managing-workflow/tuning-component-settings/)

**Comment by bacongobbler** [comment]
yes, the same issue (tuning a component) can be resolved in the same way described in #724 and #726.

**Comment by bacongobbler** [comment]
gonna close this as a duplicate as #726 so we only need to close a single ticket with the action item.
